crypto/src/main/java/org/web3j/crypto/Sign.java

```diff
@@ -36,6 +36,26 @@
             CURVE_PARAMS.getCurve(), CURVE_PARAMS.getG(), CURVE_PARAMS.getN(), CURVE_PARAMS.getH());
     static final BigInteger HALF_CURVE_ORDER = CURVE_PARAMS.getN().shiftRight(1);
 
+    static final String MessagePrefix = "\u0019Ethereum Signed Message:\n";
+
+    static byte[] getMessagePrefix(int messageLength) {
+        return MessagePrefix.concat(String.valueOf(messageLength)).getBytes();
+    }
+
+    static byte[] getMessageHash(byte[] message) {
```

Comment:
Should we call this `etEthereumMessageHash` instead? As it is an Ethereum specific method for hashing messages.

Reply:
Address in: https://github.com/web3j/web3j/pull/761/